ami/data.py

```diff
@@ -345,6 +344,120 @@ def __call__(self, raw_ts, epics_epoch=True):
         return raw_ts, int(timestamp * self.heartbeat), unix_ts
 
 
+class Timeout:
+    pass
+
+
+class TimeoutIterator:
```

Comment:
Should we copy this code into ami or add the package as dependency? I added the license/cited where it came from.